clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -3038,6 +3044,22 @@ private static void dumpIngestionState(CommandLine cmd) throws Exception {
     }
   }
 
+  private static void dumpTopicPartitionIngestionContext(CommandLine cmd) throws Exception {
+    TransportClient transportClient = null;
+    try {
+      transportClient =
+          getTransportClientForServer(getRequiredArgument(cmd, Arg.STORE), getRequiredArgument(cmd, Arg.SERVER_URL));
+      dumpTopicPartitionIngestionContext(
+          transportClient,
+          getRequiredArgument(cmd, Arg.STORE),
+          getRequiredArgument(cmd, Arg.VERSION),
+          getRequiredArgument(cmd, Arg.KAFKA_TOPIC_NAME),
```

Comment:
I am wondering that - does this command need certain certificate when it is hitting server?

Reply:
I think this is different from normal read a key from a store, more for performance monitoring and debugging purpose. So I think it is OK to go without certificate like other cmd (dump ingestion state.)